metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -9,7 +9,7 @@
       description: >
         Kibana instance UUID
     - name: name
-      type: text
+      type: keyword
```

Comment:
@ycombinator Ok? Don't see why this should be text.

Reply:
Yep, makes sense as `keyword` to me.

Reply:
I screwed up. This should remain `text`. See my more-enlightened comment here: https://github.com/elastic/beats/pull/10318/files#r250673640